### PR TITLE
update vsearch to version 1.9.7 and enable testing

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

**Comment by bgruening** [comment]
Travis is happy, anyone wants to review and merge this?

**Comment by nsoranzo** [comment]
Should all the tool versions be reset to `"@VERSION@.0"` or `"@VERSION@"` ?

File: packages/package_vsearch_1_9_7/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+categories:
+- Tool Dependency Packages
+description: VSEARCH including searching, clustering, chimera detection, dereplication, sorting, masking and shuffling of sequences.
+long_description: |
+  Open and free 64-bit multithreaded tool for processing metagenomic sequences,
+  including searching, clustering, chimera detection, dereplication, sorting, masking and shuffling.
+
+  https://github.com/torognes/vsearch
```
Comment (nsoranzo) [inline packages/package_vsearch_1_9_7/.shed.yml]: Move link to `homepage_url:`

**Comment by bgruening** [comment]
To `.0` I guess.

File: packages/package_vsearch_1_9_7/tool_dependencies.xml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <package name="vsearch" version="1.9.7">
+        <install version="1.0">
+            <actions_group>
+                <actions architecture="x86_64" os="darwin">
+                    <action type="download_file" sha256sum="34f3facaeb4a17bb70efb375f41032b96d542d19726f58e7959b7d5a6f0c069b">
+                        https://github.com/torognes/vsearch/releases/download/v1.9.7/vsearch-1.9.7-osx-x86_64.tar.gz
+                    </action>
+                    <action type="move_file" rename_to="vsearch">
```
Comment (nsoranzo) [inline packages/package_vsearch_1_9_7/tool_dependencies.xml]: `rename_to` is not needed.

File: packages/package_vsearch_1_9_7/tool_dependencies.xml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+                        <source>./bin/vsearch</source>
+                        <destination>$INSTALL_DIR/bin</destination>
+                    </action>
+                    <action type="shell_command">chmod +x $INSTALL_DIR/bin/vsearch</action>
```
Comment (nsoranzo) [inline packages/package_vsearch_1_9_7/tool_dependencies.xml]: Why not use the `chmod` action?

Comment (bgruening) [inline packages/package_vsearch_1_9_7/tool_dependencies.xml]: legacy code from the old definition ...

**Comment by nsoranzo** [comment]
:+1:

**Comment by bgruening** [comment]
Wuhu, this seems to work!

**Comment by bebatut** [comment]
@bgruening Thanks for this update
Is it possible to push it on TTS, please? Thanks. I would like to test it

**Comment by bgruening** [comment]
Sure, give me a few minutes to get a real working PC.

**Comment by bebatut** [comment]
Sure :)
Thanks

**Comment by bgruening** [comment]
Done. It's now on TTS.

**Comment by bebatut** [comment]
@bgruening Thanks !! I found it

I tried to install it on my Galaxy. But I have an installation error for `package_vsearch_1_9_7`

```
[Errno 2] No such file or directory: '/path/to/database/tmp/tmp-toolshed-mtdvf6_oh/bin/vsearch'
```

File: packages/package_vsearch_1_9_7/tool_dependencies.xml [hunk 4]
```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <package name="vsearch" version="1.9.7">
+        <install version="1.0">
+            <actions_group>
+                <actions architecture="x86_64" os="darwin">
+                    <action type="download_file" sha256sum="34f3facaeb4a17bb70efb375f41032b96d542d19726f58e7959b7d5a6f0c069b">
```
Comment (nsoranzo) [inline packages/package_vsearch_1_9_7/tool_dependencies.xml]: `s/download_file/download_by_url/` may fix @bebatut's issue.

**Comment by bgruening** [comment]
@bebatut you are not using conda ;)
Just kidding can you test the latest TTS?

**Comment by bebatut** [comment]
@bgruening I'm not using conda. But after 2 mentions of it today, I think I will have to use it ;)
TTS wrapper works. Thanks !
Do you plan to push it on TS?

**Comment by bgruening** [comment]
Pushed to MTS!

File: packages/package_vsearch_1_9_7/tool_dependencies.xml (hunk 4 above)
Comment (bgruening) [inline packages/package_vsearch_1_9_7/tool_dependencies.xml]: Jupp will push soon.

**Comment by bebatut** [comment]
Many thanks !!!

**Pull request merged**